docs/satellite-clone.md

```diff
@@ -1,41 +1,38 @@
+   - For cloning RHEL 6 backup data to a RHEL 7 machine: In addition to katello-backup created files, you also need foreman.dump and candlepin.dump.  You may utilize the included script [satellite-clone/helpers/postgresql_dump.sh](../helpers/postgresql_dump.sh) if needed.
 
-#### On the Destination node (blank host)
+2. Make sure that the blank machine has adequate space and also make sure that the root partition has all the storage space. You may utilize the included script [satellite-clone/helpers/reallocate.sh](../helpers/reallocate.sh) if needed. The ansible playbook run will fail if the free space in root partition is less than the value specified in `required_root_free_space` variable in `satellite-clone-vars.yml`
 
-1. Make sure that the Destination node has adequate space and also make sure that the root partition has all the storage space. You may utilize the included script [satellite-clone/helpers/reallocate.sh] (../helpers/reallocate.sh) if needed.
+#### Instructions ####
 
-  **Note** The ansible playbook run will fail if the free space in root partition is less than the value specified in `required_root_free_space` variable in `satellite-clone-vars.yml`
-
-2. Place the backup files in a folder on the Destination node. Also remember this folder path, as this needs to be updated in Control node config file (`backup_dir` variable in `satellite-clone-vars.yml`) later.
-
-#### On the Control node:
-
-1. Create file `satellite-clone-vars.yml` (by copying `satellite-clone-vars.sample.yml` found in the root of the project) and update it as necessary.
+1. Create file `satellite-clone-vars.yml` (by copying `satellite-clone-vars.sample.yml` found in the root of the project) and update the required variables.
 
    ```console
      # cp satellite-clone-vars.sample.yml satellite-clone-vars.yml
    ```
-2. Update the folder path of the backup files on the Destination node in `backup_dir` variable in `satellite-clone-vars.yml`.
-3. For cloning a rhel6 Satellite server to a rhel7 machine, update the variable `rhel_migration` to true.
-4. Add the IP address of the Destination node to the copied inventory file. If executing the playbook on localhost, add `ansible_connection=local` after the IP address.
-5. Run the ansible playbook:
+2. Place the backup files in a folder on the blank machine. This folder path is specified in the `backup_dir` variable in `satellite-clone-vars.yml`.
+3. Update the folder path of the backup files using the `backup_dir` variable in `satellite-clone-vars.yml`.
+4. If you are cloning RHEL 6 backup data to a RHEL 7 machine, update the variable `rhel_migration` to true in `satellite-clone-vars.yml`.
+5. Add the IP address of the blank machine to the copied inventory file and add `ansible_connection=local` after the IP address.
+   ```
+     192.168.1.12 ansible_connection=local
```

Comment:
I don't think we are using ip address anywhere other than to connect to the host.  So you can just use `localhost` as shown below:

```
  localhost ansible_connection=local
```

bonues: since this is not gonna change, you don't even need to copy and update inventory file anymore.